Homework3/task3.js

```diff
@@ -0,0 +1,8 @@
+for (let i =1; i <= 10; i++) {
+console.log(`5 * ${i} = ${5*i} `);
```

Comment:
small note - it would be better to create a variable with particular number and use it instead of hard-code.
```js
const number = 5

for (let i =1; i <= 10; i++) {
    console.log(`${number} * ${i} = ${number*i} `);
}
```
same for `while` loop